content/doc/book/managing/users.adoc

```diff
@@ -1,15 +1,129 @@
 ---
 layout: section
-wip: true
+title: Users
 ---
 ifdef::backend-html5[]
-:notitle:
-:description:
+:description: Learn how to manage users in Jenkins, including creating, configuring, and understanding user management.
 :author:
 :sectanchors:
 :toc:
+:toclevels: 4
 :hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
 endif::[]
 
 = Users
 
+Managing users in Jenkins involves creating, configuring, and maintaining user accounts for individuals who interact with the Jenkins instance. This page provides a step-by-step guide on how to manage users effectively.
+
+== What Managing Users Means
+
+In Jenkins, managing users refers to the process of creating, configuring, and maintaining user accounts. This includes:
+
+- Creating new users.
+- Configuring user details (e.g., full name, email address, password).
+- Modifying user information.
+- Deleting users.
+
+Administrators with the appropriate permissions can perform these actions.
+
+== Accessing the Manage Users Page
+
+To manage users in Jenkins, you need to have administrative privileges. Follow these steps to access the user management section:
+
+. Log in to your Jenkins instance with an account that has administrative permissions.
++
+. Navigate to the **Manage Jenkins** page from the Jenkins dashboard.
++
+image::managing/manager-users-home-page.png[Accessing Manage Jenkins page,width=600]
+
+. Click on **Users** under the **Security** section.
++
+image::managing/select-users.png[Accessing the Users section,width=600]
+
+== Creating a User
+
+To create a new user in Jenkins:
+
+. On the **Users** page, click on the **Create User** button.
```

Comment:
```suggestion
. On the **Users** page, select **Create User**.
```
Suggestion to use select instead of click. Adjust some of the wording to be more direct/[not use button](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/b/button).